HW1/Susie/question.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Registration Form</title>
+    <style>
+      p {
+        padding: 0;
+        margin: 0;
+      }
+    </style>
+  </head>
+
+  <body>
+    <form action="">
+      <h1>Community Workshop Registration</h1>
+      <fieldset>
+        <legend>Personal Detais</legend>
+        <label for="name">Name:</label>
+        <input type="text" name="name" id="name" />
```

Comment:
input box 缺少placeholder
```suggestion
        <input type="text" name="name" id="name" placeholder="Your full name"/>
```